src/modules/Dropdown/Dropdown.story.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import Dropdown, { IItem } from "./Dropdown";
+import MultipleDropdown from "./MultipleDropdown";
+import { Text } from "../Typography";
+import { useStyles } from "../../base";
+import { ThemeStyleSheetFactory } from "../../types";
+
+const stylesheet: ThemeStyleSheetFactory = theme => ({
```

Comment:
nitpick: it would be easier to perceive what items are selected if the selected items had a different background color or something 😄

Reply:
This uncovered a bug on the "selected" style  👍